### ISSUE TITLE
Authorize Updating Certs by October 24

### ISSUE THREAD

**Issue by bschuon** [vcatalano/py-authorize#85]
Hi, Authorize.net is updating it's certificates.  Will this break the library, and is there an update planned?

The article titles are:

[Entrust to DigiCert SSL Certificate Migration (KA-05545)](https://url.us.m.mimecastprotect.com/s/mA-wCNk8NohXOLRujipCy8BXG?domain=app.payment.authorize.net)

[Latest Version of Authorize.net Server-Level SSL Certificates (000003009)](https://url.us.m.mimecastprotect.com/s/CAnqCOYZOps47YGHrs8CGdkwT?domain=app.payment.authorize.net)

Unfortunately, to see them, you would need to log in to your provider account.  

Please let us know what, if anything, we can or need to do about this?

Thank you!

**Comment by robertpro** [comment]
@bschuon did you figure it out?

**Comment by bschuon** [comment]
Not really.  We are not even sure if it is an issue or not.  We use the library, but haven't took a very deep dive into how it operates.

Regards,

Bob S.
________________________________

**Comment by robertpro** [comment]
> Not really. We are not even sure if it is an issue or not. We use the library, but haven't took a very deep dive into how it operates.

Yeah, we are exactly the same.
